scala/lambdas/ingest-flow-control/src/main/scala/uk/gov/nationalarchives/ingestflowcontrol/Lambda.scala

```diff
@@ -37,28 +41,74 @@ class Lambda extends LambdaRunner[Input, Unit, Config, Dependencies] {
         flowControlConfig: FlowControlConfig,
         taskStarted: Boolean
     ): IO[Boolean] = {
-      if sourceSystems.isEmpty then IO.pure(taskStarted)
-      else if taskStarted then IO.pure(true)
+      if sourceSystems.isEmpty then
+        logger.info(Map("taskStarted" -> s"""$taskStarted"""))("list of remaining sourceSystems is empty, nothing more to do on reserved channel") >> IO.pure(taskStarted)
```

Comment:
You have a bunch of multiline strings throughout the code, but you don't need to do this; you can simply do 
`s"$taskStarted"` or in some cases, just `taskStarted.toString`